app/src/androidTest/java/ca/marshallasch/veil/database/DatabaseTest.java

```diff
@@ -22,13 +22,13 @@
 
     private Database database;
     @Before
-    public void setUp() throws Exception
+    public void setUp()
     {
         database = Database.getInstance_TETSING(InstrumentationRegistry.getTargetContext());
     }
 
     @After
-    public void tearDown() throws Exception
+    public void tearDown()
```

Comment:
Why was the exception removed?

Reply:
Because android studio says it was unused.